test/mocha.opts

```diff
@@ -1 +1,5 @@
---require buba/register
+--require ts-node/register
+--require source-map-support/register
+--full-trace
+--bail
+test/**/*-test.ts
```

Comment:
Interesting.  Is this file glob necessary now because mocha doesn't find `.ts` files automatically?

Reply:
Yeah. Nyc&mocha configuration for typescript is pretty tricky and all tutorials have such glob in `mocha.opts`.